hrjobcontract/CRM/Hrjobcontract/Page/JobContractTab.php

```diff
@@ -19,6 +19,19 @@ static function registerScripts() {
 
     $loaded = TRUE;
 
+    CRM_Core_Resources::singleton()->addVars('hrjobcontract', array(
+      'baseURL' => CRM_Core_Resources::singleton()->getUrl('org.civicrm.hrjobcontract'),
+      'contactId' => CRM_Utils_Request::retrieve('cid', 'Integer'),
+      'domainId' => CRM_Core_Config::domainID(),
+      'isLogEnabled'    => (bool) $config->logging,
```

Comment:
Could you please fix the spacing?

`'isLogEnabled' => (bool) $config->logging,`